lib/ibm_watson/visual_recognition_v3.rb

```diff
@@ -254,53 +253,52 @@ def detect_faces(images_file: nil, url: nil, images_file_content_type: nil, imag
     #   of the new classifier. Must contain a minimum of 10 images.
     #
     #   Encode special characters in the file name in UTF-8.
-    # @option args classname_positive_examples_filename [String] The filename for classname_positive_examples.
-    # @option args negative_examples_filename [String] The filename for negative_examples.
+    # @param positive_examples_filename [String] The filename for positive_examples.
+    # @param negative_examples_filename [String] The filename for negative_examples.
     # @return [DetailedResponse] A `DetailedResponse` object representing the response.
-    def create_classifier(name:, **args)
-      raise ArgumentError("name must be provided") if name.nil?
+    def create_classifier(name:, positive_examples:, negative_examples: nil, positive_examples_filename: {}, negative_examples_filename: nil)
```

Comment:
why is `positive_example_filename` {} and `negative_examples_filename` nil? Shouldn't they be consistent?

Reply:
positive_examples_filename is "mapified" (actually "hashified" when talking about ruby I guess) just like the "positive_examples" parameter.   So, we'd expect a user to set an entry in the postive_examples_filename hash using the same key as the corresponding entry in the "postiive_examples" hash.   Although in this case, the "postivie_examples_filename" parameter is not required (we'll pull the "path" from the File object set in the "positive_examples" hash if not specified).

Reply:
one more note... where you see the use of {} for a default value, that indicates that the parameter is expected to be a hash (and if fact there should be a validation for it inside the function).